app/controllers/concerns/reauthentication_required_concern.rb

```diff
@@ -38,31 +26,13 @@ def recently_authenticated?
     authn_at > Time.zone.now - IdentityConfig.store.reauthn_window
   end
 
-  def prompt_for_current_password
-    store_location(request.url)
-    user_session[:context] = 'reauthentication'
-    user_session[:factor_to_change] = factor_from_controller_name
-    user_session[:current_password_required] = true
-    redirect_to user_password_confirm_url
-  end
-
   def prompt_for_second_factor
     store_location(request.url)
     user_session[:context] = 'reauthentication'
 
     redirect_to login_two_factor_options_path(reauthn: true)
   end
 
-  def factor_from_controller_name
-    {
-      # see LG-5701, translate these
```

Comment:
Can this ticket be cancelled?

Reply:
I think so, yeah. I can do that.

Reply:
The part about the untranslated string can be cancelled, but does the copy on the page still say "sign in with your phone" for all other MFAs? If so then that part of the ticket should stay

Reply:
My hope is to remove this page since it'll be unused after this PR

Reply:
ah right!!! thx

Reply:
https://github.com/18F/identity-idp/pull/8722